core/mvc/controllers/RestController.php

```diff
@@ -126,13 +127,34 @@ class RestController extends \Phalcon\Mvc\Controller implements \Phalcon\Events\
      * @var array $aclMap
      */
     protected $aclMap = array(
```

Comment:
Wouldn't it be better to move this to the Permissions model?

Reply:
This is the REST controller's action related mapping and that should be done in RestController instead of doing It in Permissions model.